llvm/include/llvm/Passes/StandardInstrumentations.h

```diff
@@ -559,6 +559,35 @@ class DotCfgChangeReporter : public ChangeReporter<IRDataT<DCData>> {
   std::unique_ptr<raw_fd_ostream> HTML;
 };
 
+class DroppedVariableStats {
+public:
+  DroppedVariableStats() {
+    llvm::outs() << "Pass Level, Pass Name, Num of Dropped Variables, Func or "
+                    "Module Name\n";
+  };
+  // We intend this to be unique per-compilation, thus no copies.
+  DroppedVariableStats(const DroppedVariableStats &) = delete;
+  void operator=(const DroppedVariableStats &) = delete;
+
+  void registerCallbacks(PassInstrumentationCallbacks &PIC);
+
+private:
+  using VarID =
+      std::tuple<const DILocalScope *, llvm::StringRef, unsigned, unsigned>;
```

Comment:
This looks like it will conflate inlined instances of the same variable. Don't we also need an inlinedAt scope pointer?

Reply:
Also, maybe this should be a struct.